src/ui/overlay.html

```diff
@@ -42,15 +42,23 @@
 
     .item-slot span {
       position: absolute;
-      bottom: 5vh;
       color: #fff;
       background-color: rgba(0, 0, 0, 0.8);
       border-radius: 4px;
-      font-size: 17vh;
+      font-size: 12vh;
       padding: 0 5px;
       left: 50%;
       transform: translateX(-50%);
     }
+
+    #status {
+      font-size: 13vh;
+      color: red;
```

Comment:
We should use the same red tint that is used for the limiting chaos recipe instead of the pure red

Reply:
I've toned down the sepia but kept `color: red` on the error message, what do you think? 
![image](https://user-images.githubusercontent.com/4822937/71119653-a5511c00-21a8-11ea-9e92-c08a3f5577e1.png)